logs/app.log

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
You should not add your log file to github

Reply:
interestint, I added this way in ignore